libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -1079,9 +1086,13 @@ bool AP_AHRS_DCM::airspeed_estimate(float &airspeed_ret) const
 //  - otherwise from a cached wind-triangle estimate value (but returning false)
 bool AP_AHRS_DCM::airspeed_estimate(uint8_t airspeed_index, float &airspeed_ret) const
 {
+#if AP_AIRSPEED_ENABLED
```

Comment:
Need to find a better way of factoring this.

Perhaps factor out a `bool get_unconstrained_airpeed_estimate` method which can early-return?

Reply:
Done. Leaving open for you to check.